src/app/wallets/index.ts

```diff
@@ -7,6 +7,7 @@ export * from "./update-pending-invoices"
 export * from "./update-pending-payments"
 export * from "./add-invoice-for-wallet"
 export * from "./wallet-name-available"
+export * from "./reimburse-fee"
```

Comment:
Should this really be exposed outside of the module? It seems like this is a 'helper' use-case that is only called internally or?

Reply:
It also gets called from `send-payment.ts` in the `@app/lightning` module ([see here](https://github.com/GaloyMoney/galoy/pull/568/files#diff-d555811974fb87c7d88c2f0b8cb302056376a294117f01ef0a35620cb2e69ab0R383))

Do we prefer to call it more directly instead? Maybe like:
```
import { reimburseFee } from "@app/wallets/reimburse-fee"
```